feature-libs/product-configurator/rulebased/components/attribute/types/single-selection-bundle/configurator-attribute-single-selection-bundle.component.ts

```diff
@@ -31,14 +31,12 @@ export class ConfiguratorAttributeSingleSelectionBundleComponent
   @Output() selectionChange = new EventEmitter<ConfigFormUpdateEvent>();
 
   ngOnInit(): void {
-    if (this.attribute.required) {
-      if (!this.attribute.values.find((value) => value.selected)) {
-        this.onSelect(this.attribute.values[0].valueCode);
-      }
+    if (this.attribute.selectedSingleValue) {
+      this.quantity.setValue(this.attribute.quantity);
+    } else {
+      this.quantity.setValue(0);
```

Comment:
```suggestion
      this.quantity.setValue(this.attribute.selectedSingleValue ? this.attribute.quantity : 0);
```

Reply:
you can remove the whole `if` statement and replace it with ternary operator (`x ? y : z`)